api/client-server/administrative_contact.yaml

```
@@ -212,12 +231,14 @@ paths:
     post:
       summary: Begins the validation process for an email address for association with the user's account.
       description: |-
-        Proxies the Identity Service API ``validate/email/requestToken``, but
-        first checks that the given email address is **not** already associated
-        with an account on this homeserver. This API should be used to request
-        validation tokens when adding an email address to an account. This API's
-        parameters and response are identical to that of the |/register/email/requestToken|_
-        endpoint.
+        The homeserver should check that the given email address is **not**
```

I believe the implied meaning of the old text was `must`, which seems to have been lost in translation:
```suggestion
        The homeserver must check that the given email address is **not**
```